test/interpret.jl

```diff
@@ -156,3 +156,19 @@ ex = quote
 end
 frame = JuliaInterpreter.prepare_toplevel(Main, ex)
 @test JuliaInterpreter.finish_and_return!(JuliaStackFrame[], frame, true) == 1
+
+
+# "correct" line numbers
+defline = @__LINE__() + 1
+function f(x)
+    x = 2x
+    # comment
+    # comment
+    x = 2x
+    # comment
+    return x*x
+end
+frame = JuliaInterpreter.enter_call_expr(:($(f)(3)))
+JuliaInterpreter.location(frame, JuliaInterpreter.JuliaProgramCounter(1)) == defline + 1
```

Comment:
```suggestion
@test JuliaInterpreter.location(frame, JuliaInterpreter.JuliaProgramCounter(1)) == defline + 1
```

Reply:
Not entirely sure how I managed to forget the `@test`s there... Fixed now.

Reply:
One of my main motivations for writing Revise was to be able to do the following without needing to restart Julia:
- commit the new test(s)
- `git stash` the fix so the code should still be broken
- `include("runtests.jl")` and verify that they fail in the way I expected
- `git stash pop`
- `include("runtests.jl")` and verify that they pass
- amend the commit to include the fix (or just add a new one)

Otherwise I'd be doing stuff like that all the time.